File: Kotlin/alg05.kt
/* ALG05 */
fun main() {
  var X: Int
  var Y: Int
  var Z: Int
  X = readLine()!!.toInt()
  Y = readLine()!!.toInt()
  Z = Math.pow(X.toDouble(), 2.0).toInt() +
      Math.pow(Y.toDouble(), 2.0).toInt()
  println(Z)
}

File: Dart/alg05.dart
/* ALG05 */
import 'dart:io';
import 'dart:math';
void main() {
  int X;
  int Y;
  int Z;
  X = int.parse(stdin.readLineSync()!);
  Y = int.parse(stdin.readLineSync()!);
  Z = pow(X, 2).toInt() + pow(Y, 2).toInt();
  print('$Z');
}
